h2o-r/tests/testdir_jira/runit_pubdev_4659.R

```diff
@@ -0,0 +1,42 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+test_4659 <- function() {
+    prostate = read.csv(locate("smalldata/logreg/prostate.csv"))
+    prostate$our_factor<-as.factor(paste0("Q",c(rep(c(1:380),1))))
+
+    prostate<-as.h2o(prostate)
+    prostate$weight<-1
+    
+    prostate_train<-prostate[1:300,]
+    prostate_test<-prostate[301:380,]
+
+    # delete response variable from test data
+    prostate_test<-prostate_test[,-3]
+    
+    print("GLM 1")
+    model<-h2o.glm(y = "AGE", x = c("our_factor"), training_frame = prostate_train,offset_column="weight")
```

Comment:
it is better not to reuse weights for offset column

offset column is confusing to people, we shouldn't publish examples that put weight column into offset

Reply:
It is also true; I will change it. I only copied paste the test case from the JIRA without thinking about it.